config/routes.rb

```diff
@@ -3,19 +3,27 @@
   get '/', to: 'welcome#index'
 
   get '/resorts', to: 'resorts#index'
+  get '/resorts/new', to: 'resorts#new'
+  post '/resorts', to: 'resorts#create'
   get '/resorts/:id', to: 'resorts#show'
+  get '/resorts/:id/edit', to: 'resorts#edit'
+  patch '/resorts/:id', to: 'resorts#update'
+
   get '/resorts/:resort_id/pass_holders', to: 'resort_pass_holders#index'
+  #line above an extra path?????
+  get '/resorts/:resort_id/resort_pass_holders', to: 'resort_pass_holders#index'
+  get '/resorts/:resort_id/resort_pass_holders/new', to: 'resort_pass_holders#new'
+  post '/resorts/:resort_id/resort_pass_holders', to: 'resort_pass_holders#create'
 
   get '/instructors', to: 'instructors#index'
   get '/instructors/:id', to: 'instructors#show'
   get '/instructors/:instructor_id/students', to: 'instructor_students#index'
```

Comment:
I'm noticing that your updated version of routes doesn't have my recent additions of instructor and student deletes... this is probably the conflict that's coming up for this PR because I think it's the only common file we were both working on.